app/javascript/src/Policies/util.jsx

```diff
@@ -48,28 +48,12 @@ function parsePolicy (key: string, policy: RawPolicy): RegistryPolicy {
   return { ...policy, name: key, humanName: policy.name, data: {} }
 }
 
-const toJsonString = (val: Object): string => JSON.stringify(val, null, 2)
```

Comment:
Could you extract this to a different PR?

Reply:
The extracting to `utiliities/json-utils` and the import in the other files

Reply:
Done #746